src/main/java/wraith/fwaystones/mixin/FilledMapItemMixin.java

```diff
@@ -0,0 +1,22 @@
+package wraith.fwaystones.mixin;
+
+import com.llamalad7.mixinextras.injector.ModifyReceiver;
+import net.minecraft.item.FilledMapItem;
+import net.minecraft.item.map.MapState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.BlockView;
+import net.minecraft.world.WorldAccess;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import wraith.fwaystones.item.map.MapStateAccessor;
+
+@Mixin(FilledMapItem.class)
+public class FilledMapItemMixin {
+    @ModifyReceiver(method = "updateColors", at = @At(value = "INVOKE", target = "Lnet/minecraft/item/map/MapState;removeBanner(Lnet/minecraft/world/BlockView;II)V"))
```

Comment:
Can this not be done with a plain `Inject`?

Reply:
Nope, I need the original arguments from `removeBanner` in `removeWaystone` too.

Reply:
Pretty sure you could do this without using a separate library with an inject (with local capture) and a redirect. 🤔

Reply:
Resolved in [1932af9](https://github.com/LordDeatHunter/FabricWaystones/pull/132/commits/1932af93bc7ef43d058ab1efee8e829435fb8760)